src/config/holesky.json

```diff
@@ -105,5 +105,9 @@
   "uniswapPositionManager": {
     "address": "0x0000000000000000000000000000000000000000",
     "startBlock": "2673700"
+  },
+  "validatorsRegistry": {
+    "address": "0x4242424242424242424242424242424242424242",
```

Comment:
Many validators are create at the genesis without depositing to the contract. One option could be to load them from IPFS file if the file is not too large